### ISSUE TITLE
Wallet displays zero balance until name registration has completed

### ISSUE THREAD

**Issue by larrysalibra** [blockstack/blockstack-browser#549]
Via @kantai in #543 
> Another related, but different problem, with respect to wallet balances is that once a user has issued a registration, if the user only had 1 UTXO, that UTXO is used in the name registration, and the change UTXO hasn't been confirmed yet, so the user's balance will display (correctly) 0. This is more of a UX problem than a bug, and I don't have any obvious suggestions for how to indicate to a user that they're waiting on a name or a UTXO to confirm. We can move this to a separate issue if you think that's better.

**Comment by larrysalibra** [comment]
@kantai Is this because the core doesn't send the change for each registration step back to the core wallet address? If core does send the change back to, then the wallet balance *should* be correct even before it's fully confirmed.

**Comment by kantai** [comment]
Core does send the change back to the payment address on each transaction-- however, it won't display an unusable balance. Core, as currently configured, will only send out a transaction if it has inputs with >= 6 confirmations. Furthermore, even if we reconfigured it, we would also need to change the configuration of our `bitcoind.blockstack.com` server (it rejects transactions with unconfirmed inputs).

**Comment by larrysalibra** [comment]
We'll need to switch to Core rc-v0.14.3 to get this functionality.

**Comment by larrysalibra** [comment]
After updating to rc-0.14.3, this appears resolved.
